README.md

```diff
@@ -1,190 +1,97 @@
-# Header [![Build Status](https://travis-ci.org/Financial-Times/o-header.svg)](https://travis-ci.org/Financial-Times/o-header)
```

Comment:
Would it be worth adding information about the different themes available as well?